tools/cli/pkg/printer/table.go

```diff
@@ -234,19 +234,28 @@ func (t *tablePrinter) printOneObj(obj interface{}) error {
 			lastOp := *eventList[0].OperationID
 			buffer := strings.Builder{}
 			eventTabWriter := newTabWriter(&buffer)
-			printOperation(eventTabWriter, lastOp)
+			rt := obj.(runtime.RuntimeDTO)
+			opStatus := rt.Status.State
+			opType := rt.LastOperation().Type
```

Comment:
This only contains `LastOperation`, KEB needs to provide to KCP status for other operations as well.

One way to approach this is to see what `--ops` does and incorporate some of it to `--events` because there appears to be some KEB API to return more information about operations already and KCP is using it.

Another way could be to enhance the events API and [DTO](https://github.com/kyma-project/control-plane/blob/8b2cd58faf4b1a905d26806d749a1bb17ad9be16/components/kyma-environment-broker/common/events/client.go#L20-L27) and then put more information directly in the [query](https://github.com/kyma-project/control-plane/blob/8b2cd58faf4b1a905d26806d749a1bb17ad9be16/components/kyma-environment-broker/internal/storage/postsql/read.go#L687-L699) and join with `operations` table.